MsWheaPkg/Crates/RustMuTelemetryHelperLib/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "RustMuTelemetryHelperLib"

```

Comment:
If it's just "MuTelemetryHelperLib" that will be identical to the C version of the library, is that OK?

Reply:
I think so. Let me know if I'm missing something, but this would only be directly referenced by Rust code and an INF wrapper could make the distinction in the future if needed.